### ISSUE TITLE
binary exchange data

### ISSUE THREAD

**Issue by jdhughes-usgs** [MODFLOW-USGS/modflow6#128]
evaluate option to specify exchange data as binary data

    ```
    BEGIN EXCHANGEDATA
        <cellidm1> <cellidm2> <ihc> <cl1> <cl2> <hwva> [<aux(naux)>]
        <cellidm1> <cellidm2> <ihc> <cl1> <cl2> <hwva> [<aux(naux)>]
        ...
    END EXCHANGEDATA
    ```
Support this option

    ```
    BEGIN EXCHANGEDATA
        OPEN/CLOSE exchange.exc (BINARY)
    END EXCHANGEDATA
    ```

**Comment by langevin-usgs** [comment]
@mjreno, is this supported now by using IDM (Input Data Model) loading routines for exchanges (#1505)?

**Comment by mjreno** [comment]
@langevin-usgs This hasn't been tested yet but should be close if not already done. I'll find or generate a suitable input and test and then will update this issue.

**Comment by langevin-usgs** [comment]
@mjreno, I suspect this is supported now.  If so, can you close this PR?

**Comment by mjreno** [comment]
@langevin-usgs apologies for losing track of this one, will verify and close.

**Comment by langevin-usgs** [comment]
> @langevin-usgs apologies for losing track of this one, will verify and close.

No probem!  Was just looking through these issues and think some might be fixed now because of IDM.

**Comment by mjreno** [comment]
@langevin-usgs just thinking about this a bit- the struct array input reader already can read similar binary inputs.  The current use case is for stress period data, however, so currently the control record is read and binary read interface invoked only in the dynamic loader.  The change needed would be update the static loader to do the same.

If this sounds right to you it wouldn't be a lot of work on the MODFLOW 6 side- would FloPy also need to be updated to generate binary exchange list style input?

**Comment by mjreno** [comment]
Closing as this has been addressed with #1784 and #1783.
